### PR TITLE
Update Russian translation.

### PR DESCRIPTION
**Pull request by Fat-Zer**
- Add a couple translatability fixes.
- Update Russian translation.
- A minor case-related fix to csproj file.

Note: I haven't added the translations of the new strings to Spanish and Chinese.

### CODE REVIEW

File: Localization/TrajectoriesLocalization.cfg [hunk 1]
```diff
@@ -5,10 +5,12 @@ Localization
         #autoLOC_Trajectories_Title = Trajectories
         #autoLOC_Trajectories_AppButtonTooltip = Right click toggles Trajectories window
         #autoLOC_Trajectories_ConicsErr = Can't show trajectory because patched conics are not available. Please update your tracking station facility
+        #autoLOC_Trajectories_Target = Target
         #autoLOC_Trajectories_Descent = Descent
         #autoLOC_Trajectories_ShowTrajectory = Show trajectory
         #autoLOC_Trajectories_InFlight = In-Flight
         #autoLOC_Trajectories_FixedBody = Fixed body
+        #autoLOC_Trajectories_Complete = Complet
```
Comment (fat-lobyte) [inline Localization/TrajectoriesLocalization.cfg]: Complet**e** ;)

File: Plugin/MainGUI.cs [hunk 2]
```diff
@@ -23,6 +23,7 @@ public sealed class MainGUI: MonoBehaviour
         private const float slider_width = 170.0f;
         private const float integrator_slidermin = 1.0f;
         private const float integrator_slidermax = 50.0f;
+        private const float lat_long_width = 28.0f;
```
Comment (fat-lobyte) [inline Plugin/MainGUI.cs]: :+1:

File: Plugin/Trajectories.csproj [hunk 3]
```diff
@@ -80,7 +80,7 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="3rdParty\ToolbarWrapper.cs" />
-    <Compile Include="AerodynamicModel\AerodynamicModelFactory.cs" />
+    <Compile Include="AerodynamicModel\AeroDynamicModelFactory.cs" />
```
Comment (fat-lobyte) [inline Plugin/Trajectories.csproj]: Was that really not in there? How did we compile things before?

Comment (Fat-Zer) [reply Plugin/Trajectories.csproj]: Windows just doesn't care about that...

File: Localization/TrajectoriesLocalization.cfg (hunk 1 above)
Comment (Fat-Zer) [reply Localization/TrajectoriesLocalization.cfg]: fixed alongside with a couple more typos in russian...

**Comment by PiezPiedPy** [comment]
Wow, help with my crap translation so soon.
Thanks very much 😎 
I'll add the Spanish and Chinese later today,

**Comment by akors** [comment]
@Fat-Zer Cool, so are we ready to merge?

**Comment by Fat-Zer** [comment]
@akors, looks like so, good enough for me...

**Comment by fat-lobyte** [comment]
Dammit, wrong account again :roll_eyes:

Anyway, I will merge then!

**Pull request merged**